plugins/backup/fresh-backups

```diff
@@ -39,18 +39,21 @@ An example configuration snippet for backup-manager [0] follows.
 
 =head1 AUTHOR
 
-Olivier Mehani <shtrom+munin@ssji.net>
+Copyright (C) 2016,2019 Olivier Mehani <shtrom+munin@ssji.net>
 
 =head1 LICENSE
 
-GPLv2
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
I'm the original author, and the only one to have introduced meaningful changes. I hope the relicensing is OK.